templates/header.hbs

```diff
@@ -3,7 +3,10 @@
 <header class="header">
   <div class="logo">
     {{#link 'help_center'}}
-      <img src="{{settings.logo}}" alt="{{t 'home_page' name=help_center.name}}">
+      <img src="{{settings.logo}}" alt="{{t 'home_page' name=help_center.name}}" />
+      {{#if settings.show_brand_name}}
+        <span>{{help_center.name}}</span>
```

Comment:
For the home page, we have an [hidden element](https://github.com/zendesk/copenhagen_theme/blob/master/templates/home_page.hbs#L1) with the help center name as `h1` for screen readers to read it. This change would make the screen reader read it twice - not sure if it's an issue though. @VilhelmNielsen what do you think?

Reply:
Good catch, plus the alt text for the logo would make assistive technology announce the link as, eg: “Mondocam Help Center home page Mondocam.“

It's probably worth it to slap an `aria-hidden=true` on the `span` tag.